Cython/Compiler/TypeSlots.py

```diff
@@ -1036,10 +999,8 @@ def __init__(self, old_binops):
             EmptySlot("tp_getattr"),
             EmptySlot("tp_setattr"),
 
-            # tp_compare (Py2) / tp_reserved (Py3<3.5) / tp_as_async (Py3.5+) is always used as tp_as_async in Py3
-            MethodSlot(cmpfunc, "tp_compare", "__cmp__", method_name_to_slot, ifdef="PY_MAJOR_VERSION < 3"),
             SuiteSlot(self. PyAsyncMethods, "__Pyx_PyAsyncMethodsStruct", "tp_as_async",
```

Comment:
We can probably also get rid of this struct name alias now, in one of the next PRs.